site/src/scss/_base.scss

```diff
@@ -62,3 +62,5 @@ pre code {
     }
   }
 }
+
+ul li code { line-height: 2.5rem; }
```

Comment:
Good call!  But can we fix it by bringing down the padding on inline code?  It looks funny to have those super high lines.

```scss
# In _variables.scss
$code-padding: 0 .125rem;
```
Then we can just add 5px padding to pre code.